docs/source/models/other_models/alzheimers_population/index.rst

```diff
@@ -30,4 +30,230 @@
 
 ===============================
 Alzheimer's Population Model
-===============================
\ No newline at end of file
+===============================
+
+.. contents::
+  :local:
+
+Overview
+++++++++
+
+The goal of this population model is to model only simulants with
+Alzheimer's disease (and other dementias), in order to reduce the
+necessary population size for the :ref:`CSU Alzheiemer's simulation
+<2025_concept_model_vivarium_alzheimers>`. The model document is split
+into two parts: 1) initializing the population, and 2) adding new
+simulants during the simulated timeframe.
+
+Initializing the Population
++++++++++++++++++++++++++++
+
+Model Scale
+---------------------
+
+Let :math:`t_0` be the starting time of our simulation, let
+:math:`X_{t_0}` be the size of our simulated population at
+initialization (i.e., the initial population size per draw specified in
+the concept model), and let :math:`X^\text{real}_{t_0}` be the
+corresponding real-world population at time :math:`t_0` that our
+simulation is supposed to represent. The **model scale**, :math:`S`, of
+our simulation is defined to be :math:`S = X_{t_0} /
+X^\text{real}_{t_0}`. We will use the model scale
+both for initializing our simulated population and for adding new
+simulants.
+
+In our case, :math:`X^\text{real}_{t_0}` is the population of people
+with Alzheimer's disease and other dementias at time :math:`t_0` in a
+particular country. We can compute this as
+
+.. math::
+
+  X^\text{real}_{t_0} = p_\text{Alz} \cdot \hat{X}^\text{real}_{t_0},
```

Comment:
I'm tempted to change {Alz} to {AD} everywhere, because I think that is more consistent with other literature.  I'm also tempted to change \hat{X} to T (for total) because \hat has a special meaning to some statisticians.
```suggestion
  X^\text{real}_{t_0} = p_\text{AD} \cdot T^\text{real}_{t_0},
```

Reply:
Then we will end with T for total population and t for time --- @rmudambi do you think that will be as annoying as the p and P that you convinced Nathaniel to disambiguate?